tests/network-tests/src/fixtures/content/nft/englishAuctionWithExtension.ts

```diff
@@ -0,0 +1,84 @@
+import BN from 'bn.js'
+import { Api } from '../../../Api'
+import { BaseQueryNodeFixture, FixtureRunner } from '../../../Fixture'
+import { QueryNodeApi } from '../../../QueryNodeApi'
+import { Utils } from '../../../utils'
+import { IMember } from '../createMembers'
+import { PlaceBidsInAuctionFixture } from './placeBidsInAuction'
+import { assertNftOwner, assertAuctionAndBids } from './utils'
+
+export class NftEnglishAuctionWithExtensionFixture extends BaseQueryNodeFixture {
```

Comment:
This file is not actually used in this PR, or is it? I think you mistakenly commited this file that is originally introduced https://github.com/Joystream/joystream/pull/3408 . If that's the case you can remove it from this commit. It is causing compilation error anyway (see https://github.com/Joystream/joystream/runs/5553666098?check_suite_focus=true#step:4:334)

Reply:
You are right; this file is part of #3408. However, `assertAuctionAndBids` [check](https://github.com/Joystream/joystream/blob/b957cf68d0f1c66c6f0be2f9cba9989b8ba9eae2/tests/network-tests/src/fixtures/content/nft/englishAuctionWithExtension.ts#L75 ) was introduced for the NFT fixtures in this PR, So I added this file (mistakenly) again in the PR. I am removing this file, and once the PRs are merged, I will create another PR for adding the `assertAuctionAndBids` check in this fixture.